react/components/EXPERIMENTAL_Charts/LineChart/README.md

```diff
@@ -33,12 +33,12 @@ from the data to the axis.
 ```jsx noeditor
 const sampleData = require('./multilineSample').default;
 const keys = ['uv', 'pv', 'amt', 'gte', 'yaw'];
-const schema = {container: { height: 200, width: '40%'}};
+const customSchema = {container: { height: 200, width: '40%'}};
 <LineChart
   data={sampleData}
   dataKeys={keys}
-  schema={schema}
   xAxisKey='name'
+  schema={customSchema}
 />
 ```
 - Avoid to do grid competes visually with the data.
```

Comment:
Something wrong here in the sentence.

Reply:
the `to do` should be `makes`? :thinking: